Makefile

```diff
@@ -41,14 +41,14 @@ upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the pip requirements files to use the latest releases satisfying our constraints
 	pip install -qr requirements/edx/pip-tools.txt
 	# Make sure to compile files after any other files they include!
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx/pip-tools.txt requirements/edx/pip-tools.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx/coverage.txt requirements/edx/coverage.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx/paver.txt requirements/edx/paver.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx-sandbox/shared.txt requirements/edx-sandbox/shared.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx-sandbox/base.txt requirements/edx-sandbox/base.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx/base.txt requirements/edx/base.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx/testing.txt requirements/edx/testing.in
-	pip-compile --no-emit-trusted-host --no-index --upgrade -o requirements/edx/development.txt requirements/edx/development.in
+	pip-compile -v --no-emit-trusted-host --no-index --upgrade -o requirements/edx/pip-tools.txt requirements/edx/pip-tools.in
```

Comment:
why this change?

Reply:
To make debugging dependencies failures easier. It was a byproduct of failures that I had while debugging this branch.